opentelemetry-sdk/tests/metrics/test_metric_cardinality_limit.py

```diff
@@ -0,0 +1,40 @@
+
+from logging import WARNING
+from unittest import TestCase
+
+from opentelemetry.sdk.metrics import MeterProvider
+
+from opentelemetry.sdk.metrics.export import (
+    InMemoryMetricExporter,
+    PeriodicExportingMetricReader,
+)
+
+class TestMetricCardinalityLimit(TestCase):
+
+    def setUp(self):
+        self.exporter = InMemoryMetricExporter()
```

Comment:
Not sure, but if there were multiple tests in this class at some point, setUp might be vestigial now.